lib/src/pbdl/global_styles/design_systems/design_system_theme_data.dart

```diff
@@ -12,6 +14,8 @@ abstract class DesignSystemThemeData extends PBDLNode {
 
   List<DesignSystemColorScheme> colorSchemes;
 
+  int get totalMaterialCount;
```

Comment:
A suggestion here is to rename this to `totalDesignCount` so it's more generic, since in theory this could be extended to other designs 👍🏾

Reply:
You got it